packages/mermaid/src/diagrams/git/parser/gitGraph.jison

```diff
@@ -52,6 +52,7 @@ cherry\-pick(?=\s|$)                    return 'CHERRY_PICK';
 checkout(?=\s|$)                        return 'CHECKOUT';
 "LR"                                    return 'DIR';
 "BT"                                    return 'DIR';
+"TB"                                    return 'DIR';
```

Comment:
You haven't really applied direction. Take a look at `flow` and `class` diagram and see how they implement direction.

`git` graph should only have two directions for now, `LR` and `TB`.

Reply:
[Here I have implemented Direction](https://github.com/mermaid-js/mermaid/pull/4639/files#:~:text=dir%20%3D%20diagObj.db.getDirection()%3B)

Reply:
They use graphlib and set the direction. I think we can't use graphlib for gitgraph because we don't have vertices and edges and we form gitgraphs with a different logic.

Should we have to create something like graphlib for gitgraphs?